test/parallel/test-dgram-sendto.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+
+const errorMessage =
+  new RegExp(/^Error: Send takes "offset" and "length" as args 2 and 3$/);
+
+socket.bind(0);
```

Comment:
If all the `sendto()` calls are supposed to throw before actually sending, and we're verifying the error messages, I don't think binding, listening, and closing the socket is necessary.